db/snapshot-ctl.hh

```diff
@@ -27,6 +27,7 @@ namespace db {
 class snapshot_ctl : public peering_sharded_service<snapshot_ctl> {
 public:
     using skip_flush = bool_class<class skip_flush_tag>;
+    using force_view_snapshot = bool_class<class force_view_snapsot_tag>;
```

Comment:
I think that allow_view_snapshot would be better than `force`
as the latter can be interpreted as asking to take also the views' snapshot when taking the bade table snapshot.